module/localisation/RobotParticleLocalisation/src/RobotModel.hpp

```diff
@@ -112,7 +112,7 @@ namespace module::localisation {
         int getRogueCount() const {
             return n_rogues;
         }
-        Eigen::Matrix<Scalar, Eigen::Dynamic, 1> getRogueRange() const {
+        [[nodiscard]] Eigen::Matrix<Scalar, Eigen::Dynamic, 1> get_rogue() const {
             return resetRange;
```

Comment:
Is returning a range here correct?
Shouldn't you be generating a rogue here based on the rage as a uniform value (I think eigen can do it)
Also, pretty sure the return type here rather than dynamic can just be the model size * 1

Reply:
I dont know if it is correct or not. Fixing that is for a later PR (this model doesn't currently use rogues anyway so it doesn't make a difference at the moment)

Reply:
The range (in the config) is also current `(2, 2, 0)`

Reply:
Which doesn't actually make sense as a range either